src/pages/admin/Home.js

```diff
@@ -172,7 +172,7 @@ function AdminHome(props) {
           <Typography variant="h1" style={{
             color: COLORS.BIZTECH_GREEN
           }}>
-            BizTech Admins
+            BizTech FART
```

Comment:
can u undo this LOL

Reply:
i was wondering what it was cuz i never made changes to home 😂